src/app/analyze/stack/stack-details/stack-details.component.html

```diff
@@ -3,166 +3,203 @@
-            <!-- <img title="Info from metadata files. Click for more info."
-              src="icons/266-question.svg"></img> -->
-          </a>
-        </h4>
-        <div id="metadata">
+        <div class="card-pf card-pf-accented card-pf-view">
+          <h2 class="card-pf-title">
+            <a href="https://gitlab.cee.redhat.com/bayesian/Bayesian/tree/master/docs/stack-analysis.md#metadata-stack"><span class="pf pficon-help"></span><span class="card-pf-view-count"></span> Metadata</a>
+          </h2>
+          <div class="card-pf-body">
+            <div id="metadata">
           <p> Components with tests: {{componentsWithTests}} </p>
           <p> Components with dependency lock file: {{componentsWithDependencyLockFile}} </p>
-          <!-- <span *ngFor="let entry of required_enginesArr">
+          <span *ngFor="let entry of required_enginesArr">
           Required {{entry.key}} version : {{entry.value}}<br>
-        </span> -->
+        </span>
+        </div>
+            <p class="card-pf-view-notifications">
+            </p>
+          </div>
         </div>
       </div>
     </div>
 
 
-    <!-- </div> -->
-    <!-- </div> -->
     <div class="row">
       <h1> Components </h1>
 
-      <!--  <table class="table table-striped" [mfData]="componentsDataTable" #mf="mfDataTable" [mfRowsOnPage]="5">
-    <thead>
-      <tr>
-        <th style="width: 10%">
-<mfDefaultSorter by="ecosystem">Ecosystem</mfDefaultSorter>
-</th>
-<th style="width: 10%">
-<mfDefaultSorter by="package">Package</mfDefaultSorter>
-</th>
-<th style="width: 10%">
-<mfDefaultSorter by="version">Version</mfDefaultSorter>
-</th>
-<th style="width: 10%">
-<mfDefaultSorter by="latest_version">Latest Version</mfDefaultSorter>
-</th>
-<th style="width: 10%">
-<mfDefaultSorter by="public_usage">Public Usage</mfDefaultSorter>
-</th>
-<th style="width: 10%">
-<mfDefaultSorter by="relative_public_usage">Relative Public Usage</mfDefaultSorter>
-</th>
-<th style="width: 10%">
-<mfDefaultSorter by="poplarity">Popularity</mfDefaultSorter>
-</th>
-<th style="width: 10%">
-<mfDefaultSorter by="redhat_usage">Redhat Usage</mfDefaultSorter>
-</th>
-<th style="width: 10%">
-<mfDefaultSorter by="licence">Licence</mfDefaultSorter>
-</th>
-</tr>
-</thead>
-<tbody>
-  <tr *ngFor="let item of mf.data">
-    <td>{{item.ecosystem}}</td>
-    <td (click)="tdClicked(item)">{{item.package}}</td>
-    <td>{{item.version}}</td>
-    <td>{{item.latest_version}}</td>
-    <td>{{item.public_usage}}</td>
-    <td>{{item.relative_public_usage}}</td>
-    <td>{{item.popularity}}</td>
-    <td>{{item.redhat_usage}}</td>
-    <td>{{item.licence}}</td>
-  </tr>
-</tbody>
-<tfoot>
-  <tr>
-    <td colspan="4">
-      <mfBootstrapPaginator [rowsOnPageSet]="[5,10,25]"></mfBootstrapPaginator>
-    </td>
-  </tr>
-</tfoot>
-</table> -->
+      <table class="table table-striped" [mfData]="componentsDataTable" #mf="mfDataTable" [mfRowsOnPage]="5">
```

Comment:
We don't usually use the table element anymore. It is easier to get the styling and positioning we want with divs and assigning them classes. This makes it possible to build it so that it is responsive on mobile and tablet.